code/game/g_cmds.c

```diff
@@ -5048,6 +5048,37 @@ static void Cmd_MeAction_f(gentity_t* ent)
     trap_SendServerCommand(-1, va("print \"%s" S_COLOR_WHITE " %s\n\"", ent->client->pers.netname, message));
 }
 
+/*
+=================
+Cmd_My_f
+Based off of code for Cmd_Me_f
+=================
+*/
+static void Cmd_MyAction_f(gentity_t* ent)
+{
+	char message[512];
+
+	if (!ent->client)
+		return;
+
+	if (trap_Argc() < 2)
+	{
+		trap_SendServerCommand(ent - g_entities, "print \"Chat Emote Command for Owned Entities.\nUsage: type a 3rd person sentence.\nExample: 'console beeps'.\nOutput Example: 'RedShirt's console beeps'.\n\"");
+		return;
+	}
```

Comment:
the entire project has been inconsistent on style but I'm partial to 

```
if { 
  ...
}
```

even when the body is just one line. What do you think?

Reply:
That's fine with me.